src/DispatchListener.php

```diff
@@ -109,15 +112,22 @@ public function onDispatch(MvcEvent $e)
 
         $request  = $e->getRequest();
         $response = $application->getResponse();
+        $caughtException = null;
 
         try {
             $return = $controller->dispatch($request, $response);
+        } catch (\Throwable $ex) {
```

Comment:
Makes no difference as the code is the same for \Exception and \Throwable. In PHP 7+ the exceptions or error exceptions are caught by \Throwable whereas in PHP 5 the exceptions are caught by \Exception.

\Exception should be removed at some point in the future (wenn PHP5 support is dropped).